experiments/issue960/v1.py

```diff
@@ -15,9 +15,9 @@
 BENCHMARKS_DIR = os.environ["DOWNWARD_BENCHMARKS"]
 REVISIONS = ["issue960-base", "issue960-v1"]
 CONFIGS = [
-    IssueConfig("opcount-seq", ["--search", "astar(operatorcounting([state_equation_constraints()]))"]),
-    IssueConfig("diverse-pot", ["--search", "astar(diverse_potentials(random_seed=1729)"]),
-    IssueConfig("lm_count", ["--search", "astar(lmcount(lm_rhw(), admissible=true, optimal=true)"]),
+    IssueConfig("opcount-seq-lmcut-soplex", ["--search", "astar(operatorcounting([state_equation_constraints(), lmcut_constraints()], lpsolver=soplex))"]),
```

Comment:
If you re-do the experiments, I'd run them with both CPLEX and SoPlex. For the operator-counting heuristic, the config including `lmcut_constraints` is better and for the landmark heuristic, I'd also use the one with h^m landmarks (the one with `lm_merged(..., lm_hm(m=1))`).